crates/router/src/consts.rs

```diff
@@ -31,3 +31,11 @@ pub(crate) const PUB_SUB_CHANNEL: &str = "hyperswitch_invalidate";
 // Apple Pay validation url
 pub(crate) const APPLEPAY_VALIDATION_URL: &str =
     "https://apple-pay-gateway-cert.apple.com/paymentservices/startSession";
+
+// Default value for offset
+pub const DEFAULT_OFFSET: i64 = 0;
```

Comment:
No need to have this as a const. The way I see it, the logic is: "use provided offset or else start from page 1 (offset 0)", just like how your search engine functions or even SQL with the `OFFSET` clause.

In my opinion, it should be okay to directly use `0` in code like so (just providing an example):
```rust
let offset = req.offset.unwrap_or_default(); // or req.offset.unwrap_or(0);
```